lib/blocks.php

```diff
@@ -120,6 +119,7 @@ function gutenberg_reregister_core_block_types() {
 				'tag-cloud.php'                    => 'core/tag-cloud',
 				'template-part.php'                => 'core/template-part',
 				'term-description.php'             => 'core/term-description',
+				'video.php'                        => 'core/video',
```

Comment:
I think this will be the only part that needs to be backported. Then `index.php` will be automatically synced.

Reply:
@Mamaduka Sorry, where would this be backported to in core? I'm having trouble finding the corresponding spot in the wordpress-develop codebase. Or do you mean that the `video.php` file itself is what will need to be backported as `src/wp-includes/blocks/video.php`?

Reply:
Sorry, false alarm. It looks like this part is fully automated in the core package sync - https://github.com/WordPress/wordpress-develop/blob/trunk/tools/release/sync-stable-blocks.js.

Reply:
Added the label and retested flaky CI jobs. It should be good to merge after all checks pass.